cpp/ql/src/Security/CWE/CWE-311/CleartextFileWrite.ql

```diff
@@ -65,6 +65,7 @@ where
   midNode.getNode().asExpr() = mid and
   mid = w.getASource() and
   dest = w.getDest() and
+  not dest.(VariableAccess).getTarget().getName() = ["stdin", "stdout", "stderr"] and // exclude calls with standard streams
```

Comment:
Could this exclusion be done in the `isSink` predicate instead? That will reduce the number of sinks, and thus result in better performance.

Reply:
Not trivially / as cleanly, because the sink is the source of the `FileWrite` and this is the dest.  It could be done, if you have performance concerns.

Reply:
I don't really have performance concerns, no. The performance clearly won't be worse than what we have already. So let's just leave it as-is 👍.